pkg/grpc/client.go

```diff
@@ -0,0 +1,76 @@
+package grpc
+
+// gRPC client implementation to be able to connect to the gRPC server and perform the following operations:
+// - Start a remote JupyterLab server
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/cli/cli/v2/pkg/grpc/jupyter"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	requestTimeout = 30 * time.Second
+)
+
+type GrpcClient struct {
+	conn          *grpc.ClientConn
+	token         string
+	jupyterClient jupyter.JupyterServerHostClient
+}
+
+func New() *GrpcClient {
```

Comment:
rename to: `NewClient`

Callers of this pkg will use `grpc.NewClient()` which is a nice descriptor of what it is doing.